cmake/LibraryConfigurations.cmake

```diff
@@ -59,7 +59,7 @@ if (${PLATFORM} MATCHES "Desktop")
 elseif (${PLATFORM} MATCHES "Web")
     set(PLATFORM_CPP "PLATFORM_WEB")
     set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -s USE_GLFW=3 -s ASSERTIONS=1 --profiling")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} --profiling")
```

Comment:
Why was this part for web changed?
It sounds outside of the scope of this PR, I'm not sure how it's related.

Reply:
Yeah, it does not belong here actually. I made a mistake when choosing branches.

Reply:
Okay. Please drop those changes from this PR.